test/unittests/t_util.cc

```diff
@@ -792,7 +792,10 @@ TEST_F(T_Util, RfcTimestamp) {
   time_t time1 = time(NULL);
   string str = RfcTimestamp();
   strptime(str.c_str(), format, &tm);
-  time_t time2 = mktime(&tm);
+  time_t time2 = mktime(&tm) - timezone;
+  if (tm.tm_isdst > 0) {
```

Comment:
I tested it in a SLC6 setting different dates (summer and winter) and in none of the cases it enters in this new 'if', so in principle it looks like it's not necessary, but perhaps depending on the platform it could...